docs/README.md

```diff
@@ -0,0 +1,8 @@
+# ScalarDB Samples
+This repository contains sample applications for [ScalarDB](https://github.com/scalar-labs/scalardb):
+
+- [ScalarDB Sample](scalardb-sample/)
+- [ScalarDB Server Sample](scalardb-server-sample/)
+- [Multi-storage Transaction Sample](multi-storage-transaction-sample/)
+- [Microservice Transaction Sample](microservice-transaction-sample/)
+- [ScalarDB GraphQL Sample](scalardb-graphql-sample/)
```

Comment:
I guess the document site sync is like this
- `scalardb-samples/docs/README.md` -> `/README.md`
- `scalardb-samples/scalardb-sample/docs/README.md` -> `/scalardb-sample/README.md`
- `scalardb-samples/multi-storage-transaction-sample/docs/README.md` -> `/multi-storage-transaction-sample/README.md`
- ...

So, it would work?

@josh-wong Is my understanding correct?

Reply:
@brfrn169 Thanks for the heads up! Those links should point to the correct places even within GitHub. (For example, some users might prefer reading the docs in GitHub.)

In this case, since the links are not absolute URLs, maybe using a symbolic link would work here?